mlflow/R/mlflow/R/docker-utils.R

```diff
@@ -0,0 +1,214 @@
+#' Build a MLflow Docker image
+#'
+#' Build a MLflow Docker image that will an RFunc MLflow model
+#'
+#' @template roxlate-model-uri
+#' @param image_name Name of the Docker image
+#' @param port Port for serving the model (default: 8090)
+#' @param mlflow_version (Optional) Ignored if `mlflow_home` is specified,
```

Comment:
Is `(Optional)` used in the python docs? Because in R, I think this convention is very unusual so we might consider dropping it here and below?